packages/fxa-react/components/Footer/index.tsx

```diff
@@ -6,9 +6,18 @@ import React from 'react';
 import { Localized, useLocalization } from '@fluent/react';
 import LinkExternal from '../LinkExternal';
 import mozLogo from '@fxa/shared/assets/images/moz-logo-bw-rgb.svg';
+import classNames from 'classnames';
 
 export const Footer = () => {
   const { l10n } = useLocalization();
+  // This is different from the link-grey class! This one is one shade lighter
+  // and is only underlined on hover.
+  const greyFocusOutlineClasses =
+    'focus:shadow-input-grey-focus focus-visible:shadow-input-grey-focus focus-visible:outline-none';
```

Comment:
FYI: we have ticket `FXA-5741` in the backlog to remove the fuzzy focus outline on links (it was mentioned in this ticket but the ticket number was not specified). Would it make sense to either fold that work into this PR or avoid adding the fuzzy shadow border here since it’s likely to be removed soon?